Tasks/ServiceFabricDeployV1/deploy.ps1

```diff
@@ -195,6 +195,14 @@ try
 }
 catch
 {
+    if ($null -ne $Certificate)
+    {
+        $thumbprint = $Certificate.Thumbprint
```

Comment:
move this logic to helper class where we install/remove certificate. This will ensure that changes to cert store/location are well contained.